f5router/f5router_test.go

```diff
@@ -196,11 +403,20 @@ var _ = Describe("F5Router", func() {
 				routePair{"ser*es.cf.com", wildMidEndpoint},
 				routePair{"*vices.cf.com", wildBeginEndpoint},
 				routePair{"*vic*.cf.com", wild2Endpoint},
+				routePair{"broker.cf.com/1", brokerEndpoint1},
 			}
 			for _, pair := range rp {
 				up, _ = NewUpdate(logger, routeUpdate.Add, pair.url, pair.ep)
 				router.UpdateRoute(up)
 			}
+
+			// Add broker defined updates to a route without endpoint this should not yet create bigip objects
```

Comment:
Let's move the broker stuff to it's own context so we can keep testing focused.

Reply:
These should remain as is since these are testing the same API (router.UpdateRoute) and there is value in having other types of route updates being processed (non-broker route updates) at the same time since it shows that they do not interfere with one another while resources are getting created, deleted, and updated.

Reply:
Moved to its own context and configs added.